sdk/v1/client_test.go

```diff
@@ -433,7 +433,7 @@ func TestClient_PatchJob(t *testing.T) {
 
 			Convey("Then an empty ETag is returned", func() {
 				So(respHeaders, ShouldNotBeNil)
-				So(respHeaders, ShouldResemble, client.RespHeaders{ETag: ""})
+				So(respHeaders, ShouldResemble, &client.RespHeaders{ETag: ""})
```

Comment:
this should be checking for ShouldBeNil, as respHeader is nil in error scenario

Reply:
Although respHeader is nil, it is an empty RespHeaders object that gets returned by the PatchJobs function.

Reply:
ETag/Json to be nil in error scenario as per the comment above

Reply:
I have changed this in line with the GetTask method as requested